apps/omg_watcher_info/lib/omg_watcher_info/db/block.ex

```diff
@@ -54,6 +58,21 @@ defmodule OMG.WatcherInfo.DB.Block do
     DB.Repo.aggregate(__MODULE__, :max, :blknum)
   end
 
+  @doc """
+    Gets a block specified by a hash.
+  """
+  def get(hash) do
```

Comment:
General question...will blocks always be fetched by `hash`? do we know if we'll ever need to fetch blocks by `blknum`? I only ask because there is basically a foreign key relationship between `txoutputs.blknum` and `blocks.blknum`. It's not an actual DB constraint, but the relationship does exist.

If we don't know yet, let's just tuck this question away for a later time.